### DISCUSSION TITLE
how set  RenderSchedule.TransformEntryProps

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#4146] [Q&A] [answered]
in RenderSchedule script
now i want rotate some degree (for example 60 degree ) along the y direction of the element's center point( cx,cy,cz ),
how set the RenderSchedule.TransformEntryProps

**Comment by pmconne** [comment, marked as answer]
Create a [Transform](https://www.itwinjs.org/reference/core-geometry/cartesiangeometry/transform/) using any of its many static methods.
Then call [toRows](https://www.itwinjs.org/reference/core-geometry/cartesiangeometry/transform/torows/) on the Transform to obtain the 3x4 matrix needed for [TransformEntryProps.value](https://www.itwinjs.org/reference/core-common/displaystyles/renderschedule/renderschedule.transformentryprops/).
